.vitepress/config.ts

```diff
@@ -244,21 +234,45 @@ function sidebarHome() {
           ],
         },
         {
-          text: "Execution",
+          text: "Choose You Execution",
```

Comment:
```suggestion
          text: "Choose your execution environment",
```

IMO reads a little weird without "environment"

Reply:
I was also wondering if there is a better name. 
We say execution because we've been calling it the execution API, but in practice this is your `VM`. 
Like EVM, SVM, etc. 

Interested in other people's thoughts on this too.

Reply:
"Choose your VM"?